models/articles.model.js

```diff
@@ -119,8 +128,8 @@ exports.insertCommentByArticle = (id, username, body) => {
       "INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *",
       [id, username, body]
     )
-    .then(response => {
-      console.log(response)
-      return response.rows[0]; 
-    })
+    .then((response) => {
+      console.log(response);

```

Comment:
remove console log

Reply:
Think this is a different ticket. Shouldnt be making changes/committing code for a different ticket for the PR - makes it difficult to review the actual ticket